### ISSUE TITLE
do we still need the manifest file?

### ISSUE THREAD

**Issue by superfell** [superfell/Beatbox#36]
Is the manifest.in file left over from ez_setup ? or is still used somehow? if so it needs the reference to readme.md fixing.

**Comment by superfell** [comment]
@hynekcer do you know if we still need the manifest file ?

**Comment by hynekcer** [comment]
Yes. I see the difference in the zip distribution created by tox, that README is present only if the Manifest is fixed. Also [setup.py should be fixed](https://github.com/hynekcer/Beatbox/commit/2a1e7b27f).
I have no reason to learn about its replacement. I have no experience with upload to PyPI. I can not create the zip with install information, except automatically together with running tox test.

**Comment by hynekcer** [comment]
The manifest need not be repeated in itself, but the Copying can.

    -include README
    +include README.md
     include *.py
    -include MANIFEST.in
    +include COPYING
